src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -307,15 +309,19 @@ function ActivitySummary(params: {
   const isOrder = type === ActivityType.ORDER
 
   if (isOrder) {
-    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind } = activity as Order
+    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind, fulfillmentTime } =
+      activity as Order
 
     const sellAmt = CurrencyAmount.fromRawAmount(inputToken, sellAmount.toString())
     const feeAmt = CurrencyAmount.fromRawAmount(inputToken, feeAmount.toString())
     const outputAmount = CurrencyAmount.fromRawAmount(outputToken, buyAmount.toString())
+    const limitPrice = formatSmart(getLimitPrice(activity as Order))
```

Comment:
In general a good idea to store the limitPrice in both token denotations, to allow the user to switch. In the context of the sidebar, I'd omit this functionality given this interferes with the onClick event to open the GP order detail.